src/pydev_waveform.cpp

```diff
@@ -16,7 +16,8 @@
 
 #include <map>
 #include <string.h>
-
+#include <iostream>
```

Comment:
Even though it's C++ code, we're using printf() elsewhere in the project and we should stay consistent.

Reply:
Changed to use printf